include/pwm.h

```diff
@@ -129,6 +129,6 @@ int32_t pwm_set_polarity(struct pwm_desc *desc,
 
 /* Set polarity of PWM generator device */
 int32_t pwm_get_polarity(struct pwm_desc *desc,
```

Comment:
This looks like a fix, please also fix `pwmg_get_period`

Reply:
Period is fine, i think

Reply:
pwmg_ prefix ? all the others are pwm_ prefixed